Unable to use clojure-lsp-native binary with SublimeText LSP

**Describe the bug**
native binary throws an error when started from SublimeText

**To Reproduce**
Steps to reproduce the behavior:
1. Open SublimeText with LSP plugin
2. Follow configuration steps described here - https://lsp.sublimetext.io/language_servers/#clojure
3. Use `["bash", "-c", "/usr/local/bin/clojure-lsp"]` as a "command"
4. See error

**User details (please complete the following information):**
 - OS: MacOS
 - Editor: SublimeText4 
 - Version: 
   - clojure-lsp 2021.05.14-15.26.02
   - clj-kondo 2021.04.24-SNAPSHOT

**Additional context**

# Exception
```
clojure-lsp: May 21, 2021 5:22:29 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
clojure-lsp: SEVERE: Unable to invoke no-args constructor for class org.eclipse.lsp4j.GeneralClientCapabilities. Registering an InstanceCreator with Gson for this type may fix this problem.
clojure-lsp: java.lang.RuntimeException: Unable to invoke no-args constructor for class org.eclipse.lsp4j.GeneralClientCapabilities. Registering an InstanceCreator with Gson for this type may fix this problem.
clojure-lsp: 	at com.google.gson.internal.ConstructorConstructor$14.construct(ConstructorConstructor.java:228)
clojure-lsp: 	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:212)
clojure-lsp: 	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:131)
clojure-lsp: 	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:222)
clojure-lsp: 	at com.google.gson.Gson.fromJson(Gson.java:932)
clojure-lsp: 	at org.eclipse.lsp4j.adapters.InitializeParamsTypeAdapter.readCapabilities(InitializeParamsTypeAdapter.java:151)
clojure-lsp: 	at org.eclipse.lsp4j.adapters.InitializeParamsTypeAdapter.read(InitializeParamsTypeAdapter.java:109)
clojure-lsp: 	at org.eclipse.lsp4j.adapters.InitializeParamsTypeAdapter.read(InitializeParamsTypeAdapter.java:34)
clojure-lsp: 	at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:199)
clojure-lsp: 	at com.google.gson.Gson.fromJson(Gson.java:932)
clojure-lsp: 	at com.google.gson.Gson.fromJson(Gson.java:1003)
clojure-lsp: 	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.fromJson(MessageTypeAdapter.java:339)
clojure-lsp: 	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.parseParams(MessageTypeAdapter.java:298)
clojure-lsp: 	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.read(MessageTypeAdapter.java:134)
clojure-lsp: 	at org.eclipse.lsp4j.jsonrpc.json.adapters.MessageTypeAdapter.read(MessageTypeAdapter.java:55)
clojure-lsp: 	at com.google.gson.Gson.fromJson(Gson.java:932)
clojure-lsp: 	at org.eclipse.lsp4j.jsonrpc.json.MessageJsonHandler.parseMessage(MessageJsonHandler.java:119)
clojure-lsp: 	at org.eclipse.lsp4j.jsonrpc.json.MessageJsonHandler.parseMessage(MessageJsonHandler.java:114)
clojure-lsp: 	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:193)
clojure-lsp: 	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
clojure-lsp: 	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
clojure-lsp: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
clojure-lsp: 	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
clojure-lsp: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
clojure-lsp: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
clojure-lsp: 	at java.lang.Thread.run(Thread.java:829)
clojure-lsp: 	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:553)
clojure-lsp: 	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
clojure-lsp: Caused by: java.lang.reflect.InvocationTargetException
clojure-lsp: 	at java.lang.reflect.Method.invoke(Method.java:566)
clojure-lsp: 	at com.google.gson.internal.UnsafeAllocator$1.newInstance(UnsafeAllocator.java:50)
clojure-lsp: 	at com.google.gson.internal.ConstructorConstructor$14.construct(ConstructorConstructor.java:225)
clojure-lsp: 	... 27 more
clojure-lsp: Caused by: java.lang.IllegalArgumentException: Class org.eclipse.lsp4j.GeneralClientCapabilities is instantiated reflectively but was never registered. Register the class by using org.graalvm.nativeimage.hosted.RuntimeReflection
clojure-lsp: 	at com.oracle.svm.core.graal.snippets.SubstrateAllocationSnippets.hubErrorStub(SubstrateAllocationSnippets.java:261)
clojure-lsp: 	at sun.misc.Unsafe.allocateInstance(Unsafe.java:840)
clojure-lsp: 	... 30 more
```

--- a/resources/META-INF/native-image/clojure-lsp/clojure-lsp/reflect-config.json
+++ b/resources/META-INF/native-image/clojure-lsp/clojure-lsp/reflect-config.json
@@ -133,6 +133,18 @@
   "methods":[{"name":"<init>","parameterTypes":[] }]
 },
 {
+  "name":"org.eclipse.lsp4j.MarkdownCapabilities",
+  "allDeclaredFields":true,
+  "allPublicMethods":true,
+  "methods":[{"name":"<init>","parameterTypes":[] }]
+},
+{
+  "name":"org.eclipse.lsp4j.RegularExpressionsCapabilities",
+  "allDeclaredFields":true,
+  "allPublicMethods":true,
+  "methods":[{"name":"<init>","parameterTypes":[] }]
+},
+{
   "name":"org.eclipse.lsp4j.ClientInfo",
   "allDeclaredFields":true,
   "allPublicMethods":true,

